packages/jsii-pacmak/test/expected.jsii-calc-base/dotnet/Amazon.JSII.Tests.CalculatorPackageId.BasePackageId/Amazon.JSII.Tests.CalculatorPackageId.BasePackageId.csproj

```diff
@@ -24,7 +24,8 @@
     <EmbeddedResource Include="scope-jsii-calc-base-0.20.0.tgz" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Amazon.JSII.Runtime" Version="0.20.0" />
+    <PackageReference Include="Amazon.JSII.Runtime" Version="[0.20.0]" />
```

Comment:
The main issue with this is that when customers upgrade 1 dependency, they will have to upgrade them all. This is probably undesirable.

Are you sure we have to pin an exact version?

Reply:
You're right - I loosened this up to match the current major.